community/networkmanager/APKBUILD

```diff
@@ -31,9 +30,10 @@ makedepends="$depends_dev
 	util-linux-dev
 	jansson-dev
 	vala
-	autoconf
-	automake
-	libtool
+	meson
+	py-gobject3
+	libpsl-dev
+	python
```

Comment:
Better to specify exact py3 version here

Reply:
Uh? It uses py2.

Reply:
Then specify `python2`

Reply:
Ah, thanks, didn't see that!